src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -102,7 +103,7 @@ public String toString() {
     }
 
     @Override
-    public ObservableList<Person> getPersonList() {
+    public ObservableList<Question> getPersonList() {
```

Comment:
same here. maybe change from person to Question?

Reply:
edited